configure.ac

```diff
@@ -1944,14 +1921,10 @@ AS_IF([test x"$with_afl" = "xyes"],
   [afl=true],
   [afl=false])
 
-AS_IF([test x"$enable_force_safe_string" = "xno"],
-  [force_safe_string=false],
-  [AC_DEFINE([CAML_SAFE_STRING])
-   force_safe_string=true])
-
-AS_IF([test x"$DEFAULT_STRING" = "xunsafe"],
-  [default_safe_string=false],
-  [default_safe_string=true])
+# Not used internally since 5.00 but kept for compatibility
+AC_DEFINE([CAML_SAFE_STRING])
+force_safe_string=true
+default_safe_string=true
```

Comment:
In #10962 I went further and moved these kinds of things directly to `Makefile.config.in` and `s.h.in`, etc. (i.e. `FORCE_SAFE_STRING=true` directly in `Makefile.config.in`, etc.)

Reply:
Done.